### PR TITLE
Update dev setup instructions

### PR DESCRIPTION
**Pull request by mpw5**
#### What

Minor updates to the setup instructions in docs/DEVELOPMENT.md to make them more accurate and easier to follow.

#### Why

So that as a developer I have clear, accurate instructions to follow.

### CODE REVIEW

File: docs/DEVELOPMENT.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 ## Requirements
 
-* Python 3
+* Python 3.10.0
```
Comment (jrmhaig) [inline docs/DEVELOPMENT.md:5]: This now conflicts with the note, below.

File: docs/DEVELOPMENT.md [hunk 2]
```diff
@@ -71,34 +74,35 @@ By default bounds to port 8080
...
+
 ```
-cd laa-fee-calculator
-pyenv activate laa-fee-calculator
-./manage.py test
+venv/bin/python manage.py test calculator.tests.test_calculation_agfs_12
 ```
 
-run individual test
+**Alternative Python version management**
```
Comment (jrmhaig) [inline docs/DEVELOPMENT.md:89]: I haven't tried `pyenv` but you can get the same result with;

```bash
cd laa-fee-calculator
source venv/bin/activate
```

Do you know what benefit pyenv has?

(I know you have just moved this around without changing so I'm happy to approve this as it is)

File: docs/DEVELOPMENT.md (hunk 1 above)
Comment (mpw5) [reply docs/DEVELOPMENT.md:5]: Good point, I'd meant to delete that 🤦 It seems like whatever the issue was has now been resolved, so I'll delete.

File: docs/DEVELOPMENT.md (hunk 2 above)
Comment (mpw5) [reply docs/DEVELOPMENT.md:89]: I'm not really clear of the benfit of `pyenv` vs `venv`; I need to start on my Python L&D! My basic understanding is that they're two ways of doing the same thing and the instruction is just there to cater for devs who use different tools (similar to `rbenv` vs `rvm` etc in Ruby).

Maybe it's better to just remove it? This page is there to get the app running, not as a Python primer, so maybe it's overkill.

Comment (jrmhaig) [reply docs/DEVELOPMENT.md:86]: I installed `pyenv` to try this and I get;

```
pyenv: no such command `activate'
```

Maybe I'm missing something in the setup. I would agree with removing it as anyone using `pyenv` would presumably not need the instructions.

Comment (mpw5) [reply docs/DEVELOPMENT.md]: Agreed. I've removed it now 👍

**Review by jrmhaig — APPROVED** [review]

**Pull request merged**